### ISSUE TITLE
cannot got the same result after resize and normalize in djl and pytorch

### ISSUE THREAD

**Issue by jinlong-hao** [deepjavalibrary/djl#970]
## Description

I trainned a image classification model in pytorch and deploy the pt model in java with pt. when i check the result between pytorch and djl, I found that I cannot got the same result.

I  test every step and found that I cannot got the same result after resize and normalize.

My Java code：

~~~java

        Image image = ImageFactory.getInstance().fromInputStream(new FileInputStream("E:\\01-python\\02-pytorch\\kaixing\\data\\57.jpg"));
        ImageClassificationTranslator.Builder builder = ImageClassificationTranslator.builder();
        builder.addTransform( new Resize(160, 120));
        builder.addTransform(new ToTensor());
        builder.addTransform(new Normalize(new float[]{0.485f, 0.456f, 0.225f}, new float[]{0.229f, 0.224f, 0.225f} ));

        Translator<Image, Classifications> translator = builder.build();
        NDList ndList = translator.processInput(new TranslatorContext() {
            @Override
            public NDManager getNDManager() {
                return NDManager.newBaseManager();
            }
            // ...
        }, image);
        System.out.println(ndList.get(0).getShape());
        System.out.println(ndList.get(0).get(0, 0, 0));

        System.out.println(ndList.get(0).get(0, 0, 1));

        System.out.println(ndList.get(0).get(0, 0, 2));
~~~

My Java Result

~~~
(3, 120, 160)
ND: () gpu(0) float32
1.3266

ND: () gpu(0) float32
1.3517

ND: () gpu(0) float32
1.3718
~~~

My Python code 

~~~python
import torch
import torchvision
from PIL import Image
from PIL import ImageFile
ImageFile.LOAD_TRUNCATED_IMAGES = True
import time
import os
import numpy as np
transform = torchvision.transforms.Compose([
            torchvision.transforms.Resize((160, 120)),
            torchvision.transforms.ToTensor(),
            torchvision.transforms.Normalize((0.485, 0.456, 0.225), (0.229, 0.224, 0.225))
        ])
img = Image.open(r"E:\01-python\02-pytorch\kaixing\data\57.jpg")
print(transform(img))
~~~

My python result

~~~
tensor([[[ 1.3242,  1.3755,  1.3927,  ...,  0.1083,  0.2282,  0.2624],
         [ 1.3413,  1.3755,  1.3584,  ...,  0.0912,  0.1768,  0.2624],
         [ 1.3927,  1.3927,  1.3755,  ...,  0.0741,  0.1597,  0.2624],
         ...,
         [-2.1179, -2.1179, -2.1179,  ..., -1.5014, -1.5357, -1.5014],
         [-2.1179, -2.1008, -2.1008,  ..., -1.5528, -1.6384, -1.6042],
         [-2.0494, -1.9295, -1.9467,  ..., -1.5870, -1.5870, -1.6042]],

        [[ 1.3606,  1.3957,  1.4482,  ...,  0.0651,  0.2227,  0.2402],
         [ 1.3782,  1.4132,  1.4307,  ...,  0.0651,  0.1702,  0.2402],
         [ 1.4307,  1.4482,  1.4657,  ...,  0.0826,  0.1702,  0.2752],
         ...,
         [-2.0007, -2.0182, -2.0182,  ..., -1.3704, -1.4055, -1.3704],
         [-2.0007, -2.0007, -1.9832,  ..., -1.4230, -1.5105, -1.4755],
         [-1.8606, -1.7556, -1.7906,  ..., -1.4580, -1.4580, -1.4755]],

        [[ 2.2070,  2.2418,  2.3115,  ...,  1.0741,  1.1264,  1.1264],
         [ 2.2244,  2.2593,  2.2767,  ...,  1.0566,  1.0741,  1.1264],
         [ 2.2418,  2.2941,  2.3115,  ...,  1.0392,  1.0741,  1.1612],
         ...,
         [-0.9826, -1.0000, -1.0000,  ..., -0.3551, -0.3900, -0.3551],
         [-0.9477, -0.9477, -0.9651,  ..., -0.4248, -0.5294, -0.4946],
         [-0.8257, -0.7211, -0.7560,  ..., -0.4597, -0.4771, -0.4946]]])
~~~

### Expected Behavior

got the same product within java and python

## How to Reproduce?

run both java and python code with a same jpg file 

## Environment Info

java:

* djl: 0.9.0
* djl-pytorch: 1.7.0

python:

* python: 3.7
* pytorch: 1.6.0

**Comment by stu1130** [comment]
Could you try to use sum to check the tensor value? The Resize in PyTorch python is using ImageIO API, while we are using PyTorch [interpolate C++ API](https://pytorch.org/docs/master/generated/torch.nn.functional.interpolate.html). If the sum value doesn't have a huge difference. It should be fine.

**Comment by jinlong-hao** [comment]
> Could you try to use sum(input) to check the tensor value? The Resize in PyTorch python is using ImageIO API, while we are using PyTorch [interpolate C++ API](https://pytorch.org/docs/master/generated/torch.nn.functional.interpolate.html). If the sum value doesn't have a huge difference. It should be fine.

the sum result in java:  39245.32675280833
the sum result in python: 39299.0039

**Comment by stu1130** [comment]
@jinlong-hao yeah. the difference is around 0.1%. It shouldn't have much impact on prediction result

**Comment by stu1130** [comment]
close the issue as there is no pending action item. feel free to reopen the issue if you have other question
